dll/win32/browseui/shellbrowser.cpp

```diff
@@ -3681,21 +3675,14 @@ LRESULT CShellBrowser::OnOrganizeFavorites(WORD wNotifyCode, WORD wID, HWND hWnd
 
 LRESULT CShellBrowser::OnToggleStatusBarVisible(WORD wNotifyCode, WORD wID, HWND hWndCtl, BOOL &bHandled)
```

Comment:
Is this the function called from the explorer menu when you click on "show/hide status bar" (or some similarly-named menu item)? If so, have you verified whether, on Windows, when clicking on that item, it applies automatically to *all* other explorer windows as well as any new future ones that will be opened?

Reply:
In Windows Server 2003, after showing or hiding the status bar it applies to all open windows (after they navigate to a new location) and to all future explorer windows.
In ReactOS, this code only applies to new windows that opened.

Reply:
Confirmed in Win2k3.